editor/editor_help.cpp

```diff
@@ -171,7 +171,10 @@ void EditorHelp::_class_desc_resized() {
 	real_t char_width = doc_code_font->get_char_size('x').width;
 	const int display_margin = MAX(30 * EDSCALE, get_parent_anchorable_rect().size.width - char_width * 120 * EDSCALE) * 0.5;
 
-	Ref<StyleBox> class_desc_stylebox = EditorNode::get_singleton()->get_theme_base()->get_theme_stylebox("normal", "RichTextLabel")->duplicate();
+	Control *theme_base = EditorNode::get_singleton()->get_theme_base();
+	ERR_FAIL_NULL(theme_base);
```

Comment:
This will prevent a segfault but I guess it means it will raise an error too (which would still be a bug) when changing theme with several help tabs open?

Reply:
Yeah it raises this error when you close the editor:
```
ERROR: Parameter "theme_base" is null.
   at: _class_desc_resized (editor/editor_help.cpp:175)
ERROR: Parameter "theme_base" is null.
   at: _class_desc_resized (editor/editor_help.cpp:175)
```

Reply:
the crash only occur when the engine exit, the editor singleton's theme property destructed when it exit,  and `_class_desc_resized()` called after by window resized signal which might be unnecessary.
also "changing theme with several help tabs open" is working (a .gif in the comments below)

Reply:
Well this fix is correct, but it introduces a new bug, which is that closing the editor raises these errors:
```
ERROR: Parameter "theme_base" is null.
   at: _class_desc_resized (editor/editor_help.cpp:175)
ERROR: Parameter "theme_base" is null.
   at: _class_desc_resized (editor/editor_help.cpp:175)
```

So if we merge this, a new issue should be open, and that error needs to be fixed too.